frontend/components/PostList.tsx

```
@@ -0,0 +1,102 @@
+          
+          <div className="flex flex-wrap gap-2 mb-4">
+            {post.skills.map((skill, idx) => (
+              <span key={idx} className="px-2 py-1 bg-gray-100 text-gray-700 rounded-md text-xs font-medium">
+                {skill}
+              </span>
+            ))}
+          </div>
+          
+          <div className="flex justify-between items-center text-xs text-gray-500 mb-4">
+            <div className="flex items-center gap-2">
+              <svg xmlns="http://www.w3.org/2000/svg" className="h-4 w-4" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)